web/src/scripts/actions/tabActions.js

```diff
@@ -75,6 +75,51 @@ export const closeAllTabs = (containerId) => async (dispatch) => {
   dispatch({type: at.CLOSE_ALL_TABS, payload: {containerId}})
 }
 
+export const focusAdjacentTab = (containerId, iterator, options = {}) => async (dispatch, getState) => {
+  const tabs = getState().ui.tabs
+  const { groups, focusedGroupIndex } = _.get(tabs, `${containerId}`)
```

Comment:
If the container doesn't exist, this fails. `_.get` here is just shorthand for `tabs[containerId]`, so it doesn't offer any protection